### PR TITLE
oshiete.goo.ne.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/116535

<details><summary>Screenshot:</summary>

Снимок экрана 2022-04-26 в 20 56 58

Снимок экрана 2022-04-26 в 20 57 07

Снимок экрана 2022-04-26 в 20 57 34

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -40,6 +40,8 @@ hpplus.jp,tv-tokyo.co.jp,financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.j
 !
 !
 pokemongo-kouryakumatome.net##.Article__content > div[align="center"] > br
+oshiete.goo.ne.jp##div[id^="gooad-long"]
```
Comment (Alex-302) [inline JapaneseFilter/sections/specific.txt:43]: But we already have the necessary rule
image
Removed during optimization?

Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:43]: I see this rule on optimized [filter](https://filters.adtidy.org/ios/filters/7_optimized.txt).

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -40,6 +40,7 @@ hpplus.jp,tv-tokyo.co.jp,financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.j
 !
 !
 pokemongo-kouryakumatome.net##.Article__content > div[align="center"] > br
+oshiete.goo.ne.jp###q-and-a > div[style="margin-bottom: 20px; text-align:center;"]
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:43]: I guess `oshiete.goo.ne.jp##div[style="margin-bottom: 20px; text-align:center;"]` is ok.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 3]
```diff
@@ -1795,6 +1795,7 @@ edeka.de#@##smartbanner.android
 !#if (adguard_app_android || adguard_app_ios || adguard_ext_android_cb)
 kasikornbank.com###ex-app
 lazada.co.th###smartbanner-slot
+oshiete.goo.ne.jp###answer_wrapper_toggle > div.readMore > p.readMore-btn
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:1798]: `oshiete.goo.ne.jp##.readMore > p.readMore-btn` would be also ok.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 4]
```diff
@@ -1795,7 +1795,7 @@ edeka.de#@##smartbanner.android
 !#if (adguard_app_android || adguard_app_ios || adguard_ext_android_cb)
 kasikornbank.com###ex-app
 lazada.co.th###smartbanner-slot
-oshiete.goo.ne.jp###answer_wrapper_toggle > div.readMore > p.readMore-btn
+oshiete.goo.ne.jp###readMore > p.readMore-btn
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:1798]: Missed `.` after `###`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**